### PR TITLE
Chore: remove middleware logger for production environment

### PR DESCRIPTION
**Pull request by edisonjpadilla**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ dotenv.config({ path: `.env.${process.env.NODE_ENV}` });
 
 import { ValidationPipe, Logger } from '@nestjs/common';
 import { NestFactory, Reflector } from '@nestjs/core';
+import * as process from 'process';
```
Comment (sg-gs) [inline src/main.ts:7]: Nest does manage the environment, use ConfigService (or configuration as fallback) to get the environment values.

**Review by sg-gs — CHANGES_REQUESTED** [review]

File: src/main.ts (hunk 1 above)
Comment (edisonjpadilla) [reply src/main.ts:7]: I'm using the `configuration` to get the environment and validate when using the middleware logger

Comment (sg-gs) [reply src/main.ts:7]: It appears as new line on this PR, how is that?

Comment (edisonjpadilla) [reply src/main.ts]: because I did the `configuration` import below it, but anyway I removed it, I'm already only using the `configuration`

**Review by apsantiso — APPROVED** [review]

**Review by sg-gs — APPROVED** [review]

**Pull request merged**